frontend/src/Store/Actions/seriesActions.js

```diff
@@ -377,8 +380,89 @@ export const actionHandlers = handleThunks({
           });
         });
     }, MONITOR_TIMEOUT);
-  }
+  },
+
+  [UPDATE_SERIES_MONITOR]: function(getState, payload, dispatch) {
+    const {
+      id,
+      monitor
+    } = payload;
+
+    const seriesToUpdate = { id };
+    const originalepisodes = getState().episodes.items;
+
+    if (monitor !== 'None') {
+      seriesToUpdate.monitored = true;
+    }
+
+    dispatch(set({
+      section,
+      isSaving: true
+    }));
+
+    const promise = createAjaxRequest({
+      url: '/seasonPass',
+      method: 'POST',
+      data: JSON.stringify({
+        series: [
+          seriesToUpdate
+        ],
+        monitoringOptions: { monitor }
+      }),
+      dataType: 'json'
+    }).request;
+    promise.done((data) => {
+
+      dispatch(fetchEpisodes({ seriesId: id }));
 
+      const episodesToUpdate = getState().episodes.items.reduce((acc, episode) => {
```

Comment:
Since we're fetching all the episodes again we shouldn't have to update individual episodes (so 418-450 inclusive can be deleted).

Reply:
Ahh sweet, I didn't realise that - I have removed it, much tidier!